### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by Anny85-code**
### In this milestone I;
- [x] Removed current_user method in ApplicationController, since devise provided one.
- [x] Install and setup devise.
- [x] Implemented new user  registration .
- [x] Implemented User login with a combination of email and password.
- [x] Made sure Hashed password should be stored in the database.
- [x] Implemented email confirmation.
- [x] Implemented user reset password.
- [x] Modified the views in devise.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (hiromataba) [inline app/models/user.rb]: - [ ] Please add `:confirmable` to add email confirmation

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Anny85-code    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (Anny85-code) [reply app/models/user.rb]: Thanks for your review, suggested changes will be implemented.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @Anny85-code, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**